site/en/docs/android/custom-tabs/guide-partial-custom-tabs/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: "layouts/doc-post.njk"
+title: Resizing with partial Custom Tabs
+seoTitle: "Guide: resize a Custom Tab with Partial Custom Tabs"
+date: 2023-04-21
+description: Learn how to you use partial Custom Tabs for letting your users interact with your app while viewing web content. 
+authors:
+  - sebastianbenz
+---
+
+By default, Custom Tabs launch as a full-window activity. Starting in Chrome 107, you can use partial Custom Tabs to specify a different launch height in portrait mode such that users can interact with your app while viewing web content. Users can expand the Custom Tab to full-screen by dragging the toolbar handle up and restoring the initial launch height by dragging the handle down.
```

Comment:
> Starting in Chrome 107

I was initially wondering if it was worth calling out the version it was introduced, but thinking more about the version spread on mobile, is this something that should be included for all of the apis you cover?

Reply:
ideally yes